src/common/utils/__init__.py

```diff
@@ -128,9 +124,17 @@ def pretty_json(data):
     return json.dumps(data, indent=2, sort_keys=True)
 
 
-def extract_rtbh_test_stats_json(signals: Dict[str, Any]) -> Optional[str]:
+def extract_rtbh_test_stats(signals: Dict[str, Any]) -> Optional[Dict[str, float]]:
     try:
-        return signals['browserSignals']['rtbh_test_stats']
+        results = {}
+        one_run_results = json.loads(signals['browserSignals']['rtbh_test_stats'])
```

Comment:
I'm not sure if the name `one_run_results` still makes sense in the context of this function, but that's ignorable.